### PR TITLE
Update Reconstructive to the new version with ES6 Classes

### LINKED ISSUE #435: Update Reconstructive
The [Reconstructive](https://github.com/oduwsdl/Reconstructive) code has been updated to use ES6 Class syntax. We should upgrade to that.

### PR DESCRIPTION
**Pull request by ibnesayeed**
This needs to be tested well before it is merged. Once merge it will close #435.

### CODE REVIEW

File: ipwb/reconstructive.js [hunk 1]
```diff
@@ -1,116 +1,145 @@
...
-  };
+    /**
+     * A private object with varius RegExp properties (possibly derived from other properties) for internal use.
+     * 
+     * @private
+     * @type    {{urimPattern: RegExp, absoluteRefrence: RegExp, bodyEnd: RegExp}}
+     */
+    this._regexps = {
+      urimPattern: new RegExp(`^${this.urimPattern.replace('<datetime>', '(\\d{14})').replace('<urir>', '(.*)')}$`),
+      absoluteRefrence: new RegExp(`(<(iframe|a).*?\\s+(src|href)\\s*=\\s*["']?)(https?:\/\/[^'"\\s]+)(.*?>)`, 'ig'),
```
Comment (machawk1) [inline ipwb/reconstructive.js]: Should this be` absoluteReference`? It is consistently misspelled and should still work regardless.

**Comment by machawk1** [comment]
I nuked my local ipfs repo, re-`init`ed, and ran ipwb with reconstructive 0.5.0 in a new Incognito window to ensure no caching. After disconnecting my network connection and using the froggie.warc, everything appears to work as expected with the rerouting.

File: ipwb/reconstructive.js (hunk 1 above)
Comment (ibnesayeed) [reply ipwb/reconstructive.js]: That's a typo and should be fixed in the upstream as well. Thanks for pointing this out.

Comment (ibnesayeed) [reply ipwb/reconstructive.js]: Experimented with the `Co-Authored-By` for the first time. :-)

**Comment by machawk1** [comment]
@ibnesayeed Do you want to test this further before the merge?

**Comment by ibnesayeed** [comment]
> Do you want to test this further before the merge?

I am about to test it in Docker, then it should be good to go.

**Comment by ibnesayeed** [comment]
This looks good on my machine in Docker environment as well.

**Pull request merged**